src/models/event.ts

```diff
@@ -729,8 +732,10 @@ export class MatrixEvent extends EventEmitter {
      * @returns {Object} The cleartext (decrypted) content for the event
      */
     public getClearContent(): IContent | null {
-        const ev = this.clearEvent;
-        return ev && ev.content ? ev.content : null;
+        if (this.clearEvent) {
+            return this.clearEvent.content;
+        }
+        return null;
```

Comment:
the behaviour here is slightly different, where falsey things got coalesced into `null` but now they won't. For matching the signature better it would probably be good to maintain that

Reply:
Makes sense to me, made the change.

What do you think of tweaking the comment on the function? I like maintaining the old behaviour, but the comment doesn't quite match the old behaviour.

Reply:
Huh, that's a good spot. I'm actually tempted to side with the comment in that case rather than the old behaviour

Reply:
Going to move over to respecting the comment I think. The behaviour is only different in the case of falsey content, which was already being weirdly handled.